config/routes.rb

```diff
@@ -39,6 +39,7 @@
 
 
   namespace :api do
+    get 'logged_in' => 'api#logged_in'
```

Comment:
Also this url is a query, for a pure RESTafarian- 💩 head - should it be `~/session?exist` kind of thing? again, this is purely surface stuff.